src/components/account/ProfileFields.js

```diff
@@ -85,7 +85,15 @@ const ProfileFields = (props) => {
     geoInput.current.addEventListener("focusout", () => {
       setFocusAddress(false);
     });
+    geoInput.current.addEventListener("change", (e) => {
+      addressLabel.current.focus();
+    });
     addressLabel.current = geoInput.current;
+    // <!-- Just an absolute hack to get the phamtom double-parent div to hide on mobile (ruled by media query in app.css) - please forgive / improve -->
```

Comment:
@bvneilson: typo: `phamtom` should be `phantom`

Reply:
Haha! Thanks Paul!